cvtools.py

```diff
@@ -0,0 +1,174 @@
+"""
+Filename: cvtools.py
+Authors: Taruj
+Description:image processing tools for plankton images
+"""
+
+# Standard dist imports
+import os
+from math import pi
+import cv2
+from skimage import morphology, measure, restoration
+from skimage.filters import scharr, gaussian
+import numpy as np
+from scipy import ndimage
+
+
+class configuration:
```

Comment:
Import this into config/config.py 
Change all of this to snake case

example of how to use the opt
- say I want to get the ImagesPerDir from the configuration to do my preprocessing:
```
from config import opt
images_per_dir = opt.ImagesPerDir
```